web-registry/src/pages/ProjectCreate/ProjectCreate.tsx

```diff
@@ -0,0 +1,12 @@
+import { Outlet } from 'react-router-dom';
+import { CreateProjectProvider } from './ProjectCreate.context';
+
+export const ProjectCreate = (): JSX.Element => {
+  // note: we could also use react-router's built in oulet context for this.
+  // See: https://reactrouter.com/docs/en/v6/hooks/use-outlet-context
```

Comment:
another approach to this (which I like personally) is to move the data we need to share directly into this file, and use react-router's built in [helper functions](https://reactrouter.com/docs/en/v6/hooks/use-outlet-context) to manage context. Helps avoid some boilerplate